core/Model.php

```diff
@@ -44,32 +44,35 @@ public function __construct()
     }
 
     protected function fetch($sql, $params = [], $returnResultSet = 'result') {
+        $result = false;
```

Comment:
the IDE bugs that a potential void result could be returned, also, I intend on extending the exceptions to allow graceful failure, (so it would definitely make sense then!! just not there yet :D)